contribs/gmf/apps/desktop_alt/index.html.ejs

```diff
@@ -46,7 +46,8 @@
         <div class="gmf-app-content">
           <gmf-layertree
             gmf-layertree-dimensions="mainCtrl.dimensions"
-            gmf-layertree-map="::mainCtrl.map">
+            gmf-layertree-map="::mainCtrl.map"
+            gmf-layertree-isexpanded="::true">
```

Comment:
This looks to not working well...

Reply:
I don't see any errors on. What do you means ?

Reply:
=> all the group are open but the icon is on close position
open => close and open effect...

Reply:
Ah, now it's clear.
But what should we do ? The feature is not developed by Camptocamp and it was approved here: https://github.com/camptocamp/ngeo/pull/5971
Who have to debug that ?

Reply:
Why not keep it disabled?